s3-eventbridge-ecs/pattern_s3_eb_ecs.tf

```diff
@@ -0,0 +1,278 @@
+locals {
+  bucket_name     = "serverless-land-test-bucket-1233"
+  event_bus_name  = "default"
+  container_image = "975616522298.dkr.ecr.us-east-1.amazonaws.com/serverlessland-data:latest"
```

Comment:
can this container image be public?

Reply:
I have replaced those names with placeholders. The image needs to be built from task.py inside docker folder. the image will respond to Eventbridge events and send logs. So it can't be a public image.